apollos-church-api/src/data/ContentItem.js

```diff
@@ -18,6 +18,21 @@ class dataSource extends ContentItem.dataSource {
 
     return features;
   };
+
+  getChildren = async (model) => {
+    const children = await super.getChildren(model);
+
+    const filteredChildren = children.filter(
```

Comment:
this might be slightly easier if you do this at the querying/model level :D 

https://github.com/ApollosProject/apollos-cluster/blob/master/src/data/postgres-content-items/dataSource.js#L237-L238

Reply:
but don't let this block you - it's worth merging what you have now and then fixing it better down the road / never

Reply:
That was my first try, but I had trouble getting the syntax correct to get it working.